core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/dot-experiments-configuration.component.spec.ts

```diff
@@ -74,7 +74,9 @@ const defaultVmMock: ConfigurationViewModel = {
     experimentStatus: null,
     isDescriptionSaving: false,
     menuItems: null,
-    addToBundleContentId: null
+    addToBundleContentId: null,
+    isPageLocked: null,
+    dotPageRenderState: null
```

Comment:
No need assertions needed?

Reply:
just removed those props, not needed there.